src/presenter/app-presenter.js

```diff
@@ -3,40 +3,76 @@ import PointsListView from '../view/points-list-view.js';
 import PointsListItemView from '../view/points-list-item-view.js';
 import PointEditView from '../view/point-edit-view.js';
 import PointView from '../view/point-view.js';
-import { render } from '../render.js';
+import { render, replace } from '../framework/render.js';
 
 export default class AppPresenter {
-  pointsListComponent = new PointsListView();
-  pointEditFormComponent = new PointsListItemView();
+  #pointsListContainer = null;
+  #pointModel = null;
+  #pointsListComponent = new PointsListView();
+
+  #points = [];
+  #offers = [];
+  #destinations = [];
 
   constructor({ pointsListContainer, pointModel }) {
-    this.pointsListContainer = pointsListContainer;
-    this.pointModel = pointModel;
+    this.#pointsListContainer = pointsListContainer;
+    this.#pointModel = pointModel;
   }
 
   init() {
-    this.points = [...this.pointModel.getPoints()];
-    this.offers = [...this.pointModel.getOffers()];
-    this.destinations = [...this.pointModel.getDestinations()];
-
-    render(new SortView, this.pointsListContainer);
-    render(this.pointsListComponent, this.pointsListContainer);
-    render(this.pointEditFormComponent, this.pointsListComponent.getElement());
-    render(new PointEditView({
-      point: this.points[0],
-      offers: this.offers,
-      destinations: this.destinations
-    }), this.pointEditFormComponent.getElement());
-
-    for (let i = 1; i < this.points.length; i++) {
-      const pointsListItemComponent = new PointsListItemView();
-
-      render(pointsListItemComponent, this.pointsListComponent.getElement());
-      render(new PointView({
-        point: this.points[i],
-        offers: this.offers,
-        destinations: this.destinations
-      }), pointsListItemComponent.getElement());
+    this.#points = [...this.#pointModel.points];
+    this.#offers = [...this.#pointModel.offers];
+    this.#destinations = [...this.#pointModel.destinations];
+
+    this.#renderApp();
+  }
+
+  #renderPoint(point, offers, destinations) {
+    const pointsListItemComponent = new PointsListItemView();
+    const escKeyDownHandler = (evt) => {
+      if (evt.key === 'Escape') {
+        evt.preventDefault();
+        replaceFormToPoint();
+        document.removeEventListener('keydown', escKeyDownHandler);
+      }
+    };
+    const pointComponent = new PointView({
+      point,
+      offers,
+      destinations,
+      onEditClick: () => {
+        replacePointToForm();
+        document.addEventListener('keydown', escKeyDownHandler);
+      }
+    });
```

Comment:
Отделяем пробелами блоки кода
```suggestion
    const pointsListItemComponent = new PointsListItemView();
    
    const escKeyDownHandler = (evt) => {
      if (evt.key === 'Escape') {
        evt.preventDefault();
        replaceFormToPoint();
        document.removeEventListener('keydown', escKeyDownHandler);
      }
    };
    
    const pointComponent = new PointView({
      point,
      offers,
      destinations,
      onEditClick: () => {
        replacePointToForm();
        document.addEventListener('keydown', escKeyDownHandler);
      }
    });
```